torch/nn/modules/batchnorm.py

```diff
@@ -335,7 +336,7 @@ def _check_input_dim(self, input):
             )
 
 
-class BatchNorm2d(_BatchNorm):
+class BatchNorm2d(_LazyNormBase, _BatchNorm):
```

Comment:
`BatchNorm2d` shouldn't be a subclass of `_LazyNormBase` - this breaks it and is causing several test failures